src/AspNet.Security.OAuth.Keycloak/KeycloakAuthenticationOptions.cs

```diff
@@ -36,6 +37,48 @@ public KeycloakAuthenticationOptions()
             ClaimActions.MapJsonKey(ClaimTypes.Role, "roles");
         }
 
+        /// <summary>
+        /// Gets or sets the value for Keycloak client's access type
+        /// </summary>
+        public KeycloakAuthenticationAccessType AccessType { get; set; }
+
+        /// <inheritdoc />
+        public override void Validate()
+        {
+            try
+            {
+                // HACK
+                // We want all of the base validation except for ClientSecret,
```

Comment:
This makes the Task List in Visual Studio more informative.

![image](https://user-images.githubusercontent.com/1439341/136588506-bdafcd7c-4828-4f2a-89db-28706dd16d70.png)

vs.

![image](https://user-images.githubusercontent.com/1439341/136588539-74e9e7fd-933e-4d16-b8f2-86e35f25f6f0.png)


```suggestion
                // HACK We want all of the base validation except for ClientSecret,
```